kube/kubernetes_api_service.go

```diff
@@ -146,23 +147,24 @@ func (k *KubernetesApiServiceImpl) CreatePrivilegedPod(nodeName string, image st
 		VolumeMounts: volumeMounts,
 	}
 
-	hostPathType := corev1.HostPathDirectory
+	hostPathType := corev1.HostPathFile
```

Comment:
This needs to be `corev1.HostPathSocket` (https://pkg.go.dev/k8s.io/api/core/v1#HostPathType) to prevent this during Pod creation:

```
MountVolume.SetUp failed for volume "container-socket" : hostPath type check failed: /var/run/crio/crio.sock is not a file
```